README.md

```diff
@@ -49,24 +49,45 @@ like `hostnet`. If you desire that behavior, pass it the standard way like `netw
 
 ## Installation
 
+### Pip
+
 Install the latest stable version from PyPI:
 
-```
+```bash
 pip3 install podman-compose
 ```
 
 pass `--user` to install inside regular user home without being root.
 
 Or latest development version from GitHub:
 
-```
+```bash
 pip3 install https://github.com/containers/podman-compose/archive/devel.tar.gz
 ```
 
+### Homebrew
 
-or install from Fedora (starting from f31) repositories:
+```bash
+brew install podman-compose
+```
+
+### Manual
+
+```bash
+curl -o /usr/local/bin/podman-compose https://raw.githubusercontent.com/containers/podman-compose/devel/podman_compose.py
```

Comment:
podman-compose switched to main branch. Hmm, existing documentation hasn't been updated unfortunately.